### PR TITLE
Add syntax for multiple lifetimes

### LINKED ISSUE #25417: Lifetime documentation should include an example where the lifetime of a return is dependent on two arguments
The document page:
https://doc.rust-lang.org/book/lifetimes.html

For example:
fn <'a>a_or_b(a : &'a str, b : &'a str)  -> &'a str

### PR DESCRIPTION
**Pull request by steveklabnik**
Fixes #25417

### CODE REVIEW

File: src/doc/trpl/lifetimes.md [hunk 1]
```diff
@@ -134,8 +134,29 @@ x: &'a i32,
...
 
-uses it. So why do we need a lifetime here? We need to ensure that any
-reference to the contained `i32` does not outlive the containing `Foo`.
+uses it. So why do we need a lifetime here? We need to ensure that any reference
+to a `Foo` cannot outlive the reference to an `i32` it contains.
+
+If you have multiple references, you can use the same lifetime multiple times:
+
+```rust
+fn x_or_y<’a>(x: &‘a str, y: &‘a str) -> &‘a str {
```
Comment (alexcrichton) [inline src/doc/trpl/lifetimes.md]: I think this is using the fancy single quote instead of the normal one?

Comment (alexcrichton) [inline src/doc/trpl/lifetimes.md]: (same below)

**Comment by alexcrichton** [comment]
r=me with just a minor nit

File: src/doc/trpl/lifetimes.md (hunk 1 above)
Comment (steveklabnik) [inline src/doc/trpl/lifetimes.md]: ugh i am bad at git, fixing

**Pull request merged**